### DISCUSSION TITLE
Which do you prefer [Kinoko no Yama] or [Takenoko no sato]?

### DISCUSSION THREAD

**Discussion by kodaiwasaki** [NAIST-SE/AttractingContributors#888] [2023 Attraction Contributions]
Let's put an end to this war.
この戦いに終止符を打ちましょう．

https://github.com/kodaiwasaki/AttractingContributors_takenokosaiko

**Comment by ALORAMFER** [comment]
Where is the link of your repository?

**Comment by carlos-paniagua** [comment]
@kodaiwasaki 
What are you talking about
